src/layman/util.py

```diff
@@ -67,6 +67,8 @@ def decorated_function(*args, **kwargs):
 def check_username(username):
     if not re.match(USERNAME_RE, username):
         raise LaymanError(2, {'parameter': 'user', 'expected': USERNAME_RE})
+    if username == settings.LAYMAN_GS_USER:
```

Comment:
I would slightly prefere if this check is done by geoserver modul.

Reply:
Makes sense, thanks